renovate.json

```diff
@@ -18,6 +18,13 @@
 	"major": {
 		"dependencyDashboardApproval": true
 	},
+	"docker": {
+		"fileMatch": [
+			"(^|/)Earthfile$",
+			"(^|/|\\.)Dockerfile$",
+			"(^|/)Dockerfile[^/]*$"
+		]
+	},
```

Comment:
That should help Renovate keep the Alpine version up-to-date, see https://github.com/renovatebot/renovate/discussions/24740#discussioncomment-7154873.